ROADMAP.md

```diff
@@ -41,23 +17,22 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
   * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
   * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
   * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
-  * Incorporate new environments [#1315](https://github.com/rook/rook/issues/1315)
-  * Incorporate more architectures [#1901](https://github.com/rook/rook/issues/1901)
-* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
-* CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
-  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
-* Minio
-  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
 * Support for dynamic provisioning of new storage types
   * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
   * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
+* Cassandra
```

Comment:
all these Cassandra links are to pull requests, not to issues.  Some actually redirect to issues, so I think some of them might be the right numbers but just hitting the pull REST API instead of issues API.  We should clean that up.
https://github.com/rook/rook/pull/2365 should be https://github.com/rook/rook/issues/2363 though.

Reply:
yeah those should be issues, not PRs. copy-paste bug. good catch!